.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,367 @@
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                cd test/scale 
+                chmod +x label-nodes.sh
+                ./label-nodes.sh
+            name: "LabelNodes"
+            displayName: "Label all Nodes"
+  - stage: scale_cluster_deployments
+    displayName: "Scale deploments for Network Policies Check"
+    jobs:
+      - job: scale_deployments
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                echo "collect cpu and memory usage before scaling for network policies"
+                mkdir before_netpol_cpu_and_mem
+                cd before_netpol_cpu_and_mem
+                echo "running k top node" 
+                kubectl top node >> "node_before_netpol_scale.log"
+                echo "running k top pod"
+                kubectl top pod -A | grep cilium >> "pod_before_netpol_scale.log"
+                echo "Logs will be available as a build artifact"
+                ARTIFACT_DIR=$(Build.ArtifactStagingDirectory)/before_netpol_cpu_and_mem/
+                echo $ARTIFACT_DIR
+                sudo rm -rf $ARTIFACT_DIR 
+                sudo mkdir $ARTIFACT_DIR
+                cd ..
+                sudo cp ./before_netpol_cpu_and_mem/* $ARTIFACT_DIR
+                echo "scale deployment and to prep for network policies test"
+                cd test/scale
+                chmod +x test-scale.sh
+                ./test-scale.sh --max-kwok-pods-per-node=0 --num-kwok-deployments=0 --num-kwok-replicas=0 --max-real-pods-per-node=${REAL_PODS_PER_NODE_NETPOL} --num-real-deployments=${NUM_REAL_DEPLOYMENTS_NETPOL} --num-real-replicas=${NUM_REAL_REPLICAS_NETPOL} --num-network-policies=${APPLIED_NETPOL} --num-unapplied-network-policies=${UNAPPLIED_NETPOL} --num-unique-labels-per-pod=0 --num-unique-labels-per-deployment=5 --num-shared-labels-per-pod=3 --num-real-services=${NUM_REAL_SVC_NETPOL} --delete-labels
+            name: "scaling"
+            displayName: "Run scale script"
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: before_netpol_cpu_and_mem
+              pathtoPublish: "$(Build.ArtifactStagingDirectory)/before_netpol_cpu_and_mem"
+            condition: always()
+            name: "PublishResults"
+            displayName: "Result Artifacts"
+  - stage: test_network_policies_connectivity
+    displayName: "Test Network Policies"
+    jobs:
+      - job: network_policies
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        timeoutInMinutes: 120
```

Comment:
Same here

Reply:
about 1 hour, but I adjusted the timeouts in case it goes a little over, or if we want to change the test input with different cases